pkg/controllers/management/auth/project_cluster_handler.go

```diff
@@ -3,6 +3,8 @@ package auth
 import (
 	"reflect"
 
```

Comment:
Not a big deal, but if you have to edit this PR gain, could you remove this empty line so that fmt and reflect are in the same block of imports? All standard lib imports should be grouped together at the top

Reply:
updated